tensorzero-internal/src/endpoints/inference.rs

```diff
@@ -457,7 +457,9 @@ fn create_stream(
             match chunk {
                 Ok(chunk) => {
                     buffer.push(chunk.clone());
-                    yield Ok(prepare_response_chunk(&metadata, chunk));
+                    if let Some(chunk) = prepare_response_chunk(&metadata, chunk) {
```

Comment:
Do we eventually plan to support streaming thought chunks back to the user?

Reply:
see e.g. the tests in `reasoning.rs`

Reply:
Sorry, I meant json functions (since `JsonInferenceResponseChunk` doesn't contain thought information)